build.gradle

```diff
@@ -127,6 +127,7 @@ project(':ambry-server') {
     dependencies {
         compile project(':ambry-clustermap'),
                 project(':ambry-coordinator'),
+                project(':ambry-router'),
```

Comment:
think this can be unit test compile only - since the production code doesn't depend on router.